src/modules/Channel/components/SuggestedMentionList/index.tsx

```diff
@@ -56,7 +56,17 @@ function SuggestedMentionList(props: SuggestedMentionListProps): JSX.Element {
   const [searchString, setSearchString] = useState('');
   const [lastSearchString, setLastSearchString] = useState('');
   const [currentUser, setCurrentUser] = useState<User>(null);
-  const [currentMemberList, setCurrentMemberList] = useState<Array<Member>>([]);
+  const [currentMemberList, setCurrentMemberList] = useState<Member[]>([]);
+
+  function getCurrentSearchedMemberList(): Member[] {
```

Comment:
does this have to useCallback @HoonBaek ?

Reply:
Yeah, good to have

Reply:
Oh I just left a comment here https://github.com/sendbird/sendbird-uikit-react/pull/832#discussion_r1381209395 about this.